src/network/clientpackethandler.cpp

```diff
@@ -1327,7 +1327,8 @@ void Client::handleCommand_HudSetStars(NetworkPacket *pkt)
 	StarParams stars;
 
 	*pkt >> stars.visible >> stars.count
-		>> stars.starcolor >> stars.scale;
+		>> stars.starcolor >> stars.scale
+		>> stars.visible_at_day;
```

Comment:
What happens on old servers + new clients? This will probably throw an exception, unless the server sends a too large packet.

Reply:
You probably just need to wrap this in a try-catch like so:

```cpp
    try {
        *pkt >> stars.visible_at_day;
    } catch (SerializationError &e) {};
```

But please test that is actually works.